FRAMEWORKS.yml

```diff
@@ -156,7 +156,7 @@ scala:
     version: "10.0"
   http4s:
     website: http4s.org
-    version: "0.0"
+    version: "0.18.17"
```

Comment:
Could you please specify X.Y format (e.g : 0.18)

It is more flexible, since modifying this file (and regenerate results) after each patch upgrade, will be hard to maintain